content/docs/community/automated-testing.md

```diff
@@ -5,46 +5,37 @@ description: 'Our automated testing strategy and resources'
 
 ## Terraform Testing
 
-Nearly all of our Terraform modules are updated with automated tests. We have 2 general strategies. First is by running some [`bats`](https://github.com/bats-core/bats-core) tests that do some basic sanity checks.
-These tests are defined in our [`test-harness`](https://github.com/cloudposse/test-harness) and generally don't require any credentials to run. Since that's the case, they can really only do basic linting and static analysis.
+All of our Terraform modules have automated tests. We have two sets of checks: 
 
-Then we have some tests which are based on the [`terratest`](https://github.com/gruntwork-io/terratest) library for infrastructure testing that do more in-depth integration tests of module functionality.
+- First set is running github shared workflow [`feature-branch`](https://github.com/cloudposse/github-actions-workflows-terraform-module/blob/main/.github/workflows/feature-branch.yml) 
+which does basic sanity checks: linting, formatting and docs. These checks are cheap to run and do not require any special permissions, so we run them automatically on every commit.
+You also can and should run this set of checks locally using `make precommit/terraform` command on your host before committing your changes locally and pushing your PR. You should run them because, among other things, they update the automatically generated documentation, and if you do not do it on your host, your PR will be blocked until you accept the changes via PR suggestions.
```

Comment:
~~~suggestion
Before committing and pushing your changes, you can and should run this set of checks locally by executing the following command on your host
```
make precommit/terraform
```
Running these checks locally incorporates all the required changes that otherwise would block your PR.
~~~